utils/api/services.js

```diff
@@ -12,10 +12,10 @@ export const useAllWares = (accessToken) => {
 }
 
 export const useFilteredWares = (query, accessToken) => {
-  const { data, error } = useSWR([`/providers/${process.env.NEXT_PUBLIC_PROVIDER_ID}/wares.json?q=${query}`, accessToken])
+  const { data, error } = useSWR([`/wares.json?q=${query}`, accessToken])
 
   return {
-    wares: data?.ware_refs,
+    wares: data?.ware_refs.filter(item => item.slug !== 'make-a-request'),
```

Comment:
the default ware shows with all the others. we are hiding it from the "browse" page. it will show up instead on the "requests" page.